src/Upgrade/Application/Dto/StepsResponseDto.php

```diff
@@ -239,23 +245,27 @@ public function setPullRequestId(?int $pullRequestId)
     }
 
     /**
-     * @param string $blockerInfo
+     * @param \Upgrade\Application\Dto\ValidatorViolationDto $blockerInfo
      *
-     * @return $this
+     * @return void
      */
-    public function setBlockerInfo(string $blockerInfo)
+    public function addBlocker(ValidatorViolationDto $blockerInfo): void
     {
-        $this->blockerInfo = $blockerInfo;
+        $currentReleaseGroupId = (int)$this->currentReleaseGroupId;
```

Comment:
```
php > var_dump((int) null);
int(0)
```

maybe, some additional check to avoid requesting 0th element from the `$this->blockers` ?